common/men_at_arms_types/00_maa_types.txt

```diff
@@ -423,6 +423,35 @@ massive_aerial_unit = {
 	
 }
 
+riflemen = {
+	type = archers
+	
+	damage = 65
+	toughness = 20
+	pursuit = 20
+	screen = 0
+	
+	terrain_bonus = {
+		jungle = { damage = -15 }
+		forest = { damage = -10 }
+		hills = { damage = 10 }
+	}
+
+	counters = {
+		heavy_infantry = 1
+		heavy_cavalry = 1
+		massive_aerial = 1
+	}
+
+	buy_cost = { gold = riflemen_recruitment_cost }
+	low_maintenance_cost = { gold = riflemen_low_maint_cost }
+	high_maintenance_cost = { gold = riflemen_high_maint_cost }
+	icon = gunmen
+	
+	stack = 100
+	ai_quality = { value = 64.3 }
```

Comment:
Could you make it a new value in `wc_men_at_arms_values.txt` to follow the same pattern other units use?